sdk/mixedreality/azure-mixedreality-authentication/azure/mixedreality/authentication/_shared/aio/mixedreality_account_key_credential.py

```diff
@@ -0,0 +1,54 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from datetime import date, datetime
+
+from azure.core.credentials import AccessToken
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any
+    from azure.core.credentials import AzureKeyCredential
+
+ACCOUNT_KEY_VALID_YEARS = 10
```

Comment:
Is this a good default? It seems like it would allow tokens to remain valid essentially as long as the account key is valid. I would expect tokens to have a relatively short life so that if they were intercepted the damage would be limited.

Reply:
Take a look at the comment below. This is ONLY for account keys. It's not for the token retrieved using an account key. There is no expiration for account keys.